Source/WTF/wtf/Assertions.h

```diff
@@ -725,7 +725,7 @@ static constexpr bool unreachableForValue = false;
 #define LOG_ANDROID_SEND(channel, priority, fmt, ...) do { \
     auto& logChannel = LOG_CHANNEL(channel); \
     if (logChannel.state != WTFLogChannelState::Off) \
-        __android_log_print(ANDROID_LOG_ ## priority, LOG_CHANNEL_WEBKIT_SUBSYSTEM, "[%s] " fmt, logChannel.name, ##__VA_ARGS__); \
+        __android_log_print(ANDROID_LOG_ ## priority, LOG_CHANNEL_WEBKIT_SUBSYSTEM, "[%s] " fmt, logChannel.name, SAFE_PRINTF_TYPE(__VA_ARGS__)); \
```

Comment:
I think this should use `__VA_OPT__` to prevent the expansion of `SAFE_PRINTF_TYPE()` when only the `fmt` macro parameter is passed:

 ```suggestion
        __android_log_print(ANDROID_LOG_ ## priority, LOG_CHANNEL_WEBKIT_SUBSYSTEM, "[%s] " fmt, logChannel.name, __VA_OPT__(, SAFE_PRINTF_TYPE(__VA_ARGS__))); \
```

This is the same way it's done in the `SAFE_[SF]PRINTF()` macros in `StdLibExtras.h`.